AirCasting/CreateSessionViews/PlacePicker.swift

```diff
@@ -0,0 +1,60 @@
+// Created by Lunar on 11/08/2021.
+//
+
+import SwiftUI
+import GooglePlaces
+
+struct PlacePicker: UIViewControllerRepresentable {
+    @EnvironmentObject var tracker: LocationTracker
+    @Environment(\.presentationMode) var presentationMode
+    @Binding var address: String
+
+    func makeUIViewController(context: UIViewControllerRepresentableContext<PlacePicker>) -> GMSAutocompleteViewController {
+        GMSPlacesClient.provideAPIKey(GOOGLE_PLACES_KEY)
+        
+        let autocompleteController = GMSAutocompleteViewController()
+        autocompleteController.delegate = context.coordinator
+
+        let fields: GMSPlaceField = GMSPlaceField(rawValue: UInt(GMSPlaceField.name.rawValue) | UInt(GMSPlaceField.coordinate.rawValue))
+        autocompleteController.placeFields = fields
+
+        let filter = GMSAutocompleteFilter()
+        filter.type = .address
+        filter.type = .geocode
```

Comment:
you assign to `type` twice here. I think this will be equivalent to just `filter.type = .geocode` ?